<<<<<<< HEAD
string data
=======
string message
>>>>>>> 8c9c43e (Correct changes)
---
bool success
